submitit/core/core.py

```diff
@@ -257,10 +257,26 @@ def cancel(self, check: bool = True) -> None:
             [self._cancel_command, f"{self.job_id}"], shell=False
         )
 
+    def _extract_single_result(self, results) -> R:
```

Comment:
Not sure this is worth it's own function

Reply:
it was mostly for the raise :)